### PR TITLE
Fixes btcsuite/btcd#1653

### PR DESCRIPTION
**Pull request by vinayakb**

### CODE REVIEW

File: btcjson/chainsvrresults.go [hunk 1]
```diff
@@ -108,6 +108,7 @@ type GetBlockVerboseTxResult struct {
 	VersionHex    string        `json:"versionHex"`
 	MerkleRoot    string        `json:"merkleroot"`
 	Tx            []TxRawResult `json:"tx,omitempty"`
+	RawTx         []TxRawResult `json:"rawtx,omitempty"` // Note: this field is always empty when verbose != 2.
```
Comment (onyb) [inline btcjson/chainsvrresults.go:111]: Can you also mention that the field is deprecated in Bitcoin Core, as of at least 0.16.0?

**Review by onyb — APPROVED** [review]
Looks good! Added a small suggestion to update the comment.

**Comment by vinayakb** [comment]
@onyb Updated based on your comment.

**Review by jakesyl — COMMENTED** [review]
LGTM!

**Review by jakesyl — APPROVED** [review]

**Pull request merged**